api/src/ws/game.ts

```diff
@@ -133,11 +113,29 @@ export const message: IMessage<Promise<void>> = async (socket, message) => {
 
     case 'move': {
       if (socket.cn === room.turn) {
-        refreshTime(room, player);
-        room.turn = room[opponent].name;
 
-        const gameInfo = getGameInfo(room);
-        socket.publish(socket.url, JSON.stringify({ type: 'move', data: gameInfo }));
+        const coordinates = {
```

Comment:
Why are you introducing another term here? 
```suggestion
        const board = {
```

Reply:
Because it's not a board. It's the coordinates. The coordinates are sent to the client which will change the board (depending on the result). See /client/src/lib/game.js line 39.
The difference is that a board is an object with coordinates as its keys, containing data about the piece. The coordinates variable you quoted on the other hand are the key names.
```
// board
const board = { A1: {...someData}, B1: {...someData} };

// coordinates
const coordinates = { origin: 'A1', destination: 'B1' };
```